gnomad/utils/annotations.py

```diff
@@ -2406,61 +2169,47 @@ def _create_group_dicts(
             list_of_group_info_dicts.append(group_result)
 
     # Overall frequency, via label 'adj' which is currently stored at
-    # position #1 (index 0)
-    overall_freq = ht.freq[0]
+    # position #1 (index 0).
+    overall_freq = input_struct.freq[0]
 
-    # Final dictionary to be returned
-    final_freq_dict = hl.struct(
-        **{
-            "id": hl.format("%s-%s:%s", label_name, label_version, ht.gnomad_id),
-            "type": "CohortAlleleFrequency",
-            "label": hl.format("Overall Cohort Allele Frequency for %s", ht.gnomad_id),
-            "derivedFrom": {
-                "id": f"{label_name}{label_version}",
-                "type": "DataSet",
-                "label": f"{label_name} v{label_version}",
-                "version": f"{label_version}",
-            },
-            "focusAllele": "",  # TODO load from vrs_json table
-            "focusAlleleCount": overall_freq["AC"],
-            "locusAlleleCount": overall_freq["AN"],
-            "alleleFrequency": overall_freq["AF"],
-            "cohort": {"id": "ALL"},
-        }
-    )
+    # Final dictionary to be returned.
+    final_freq_dict = {
+        "id": f"{label_name}-{label_version}-{gnomad_id}",
+        "type": "CohortAlleleFrequency",
+        "label": f"Overall Cohort Allele Frequency for {gnomad_id}",
+        "derivedFrom": {
+            "id": f"{label_name}{label_version}",
+            "type": "DataSet",
+            "label": f"{label_name} v{label_version}",
+            "version": f"{label_version}",
+        },
+        "focusAllele": "",  # TODO load from VRS dictionary
```

Comment:
remaining TODO?

Reply:
ah, the TODO means that it is filled in later inside of `gnomad.py` , but then again that probably isn't the best usage maybe? Would there be a better way to say "this is empty now but it is filled in later, dont worry" or does that even need said

Reply:
TODO is usually used in python scripts to indicate something that still needs to be worked on (https://medium.com/imdoneio/the-imdone-pitch-feedback-welcome-386430accf01). Can comment "information for this field can be populated with the result of add_gks_vrs()", and also note this in the docstring for the function